.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
I added this file to ignore files and directories not to be included in the source control (github)

Reply:
check what gitignore file is and why it is used